lib/usdUfe/ufe/UsdCamera.h

```diff
@@ -90,6 +90,9 @@ class USDUFE_PUBLIC UsdCamera : public Ufe::Camera
     Ufe::ProjectionUndoableCommand::Ptr projectionCmd(Ufe::Camera::Projection projection) override;
     Ufe::Camera::Projection             projection() const override;
 
+#ifdef UFE_V6_FEATURES_AVAILABLE
+    bool              renderable() const override;
+#endif
```

Comment:
Having the UFE_V6 is not correct. This method was only added in a specific v0.6.x version. Have a look at https://github.com/Autodesk/maya-usd/blob/dev/cmake/modules/FindUFE.cmake#L202 which is an example of where we look for the new method in the header file and then set a cmake var. Search for that cmake var and you'll see where it gets turned into a define for the h/cpp and an env var for the tests. A test is something you need to add as well. You can add something to the existing testCamera.py